src/components/WalletAssets.vue

```diff
@@ -121,11 +111,31 @@ export default class WalletAssets extends Mixins(LoadingMixin, FormattedAmountMi
   @state.account.withoutFiatAndApy private withoutFiatAndApy!: boolean;
   @state.settings.shouldBalanceBeHidden private shouldBalanceBeHidden!: boolean;
   @state.settings.permissions permissions!: WalletPermissions;
+  @state.settings.filters filters!: WalletAssetFilters;
```

Comment:
I don't see `filters` in html template above and somewhere externally, so, maybe it's to make it private